sources/packages/backend/apps/db-migrations/src/sql/Queue/Update-cas-supplier-integration.sql

```diff
@@ -0,0 +1,12 @@
+UPDATE
+    sims.queue_configurations
+SET
+    queue_configuration = '{
+        "cron": "0 19 * * 1-5",

```

Comment:
AC says noon. Either we can have the AC updated or change here. 

![image](https://github.com/user-attachments/assets/a9cd4c3c-2c96-4189-8193-3159ae337d77)

Reply:
My bad. I missed the PT part. As we discussed noon PST should be 8PM UTC.

Reply:
My bad too. All schedulers on wiki are using PST as a reference.